packages/charts/src/components/accessibility/partition_data_table.tsx

```diff
@@ -90,7 +90,7 @@ const ScreenReaderPartitionTableComponent = ({
             .slice(0, rowLimit)
             .map(({ panelTitle, depth, label, parentName, valueText, percentage }, index) => {
               return (
-                <tr key={`row--${index}`} ref={rowLimit === index ? tableRowRef : undefined} tabIndex={-1}>
+                <tr key={String(`row--${index}`)} ref={rowLimit === index ? tableRowRef : undefined} tabIndex={-1}>
```

Comment:
This should not be needed.

```suggestion
                <tr key={`row--${index}`} ref={rowLimit === index ? tableRowRef : undefined} tabIndex={-1}>
```

Reply:
Fixed in 51fd846 thank you!